.github/actions/javascript/awaitStagingDeploys/index.js

```diff
@@ -390,11 +393,13 @@ class GithubUtils {
                     });
                 }
 
+                // Internal QA PR list
                 if (!_.isEmpty(internalQAPRMap)) {
+                    console.log('Found the following verified Internal QA PRs:', resolvedInternalQAPRs);
                     issueBody += '\r\n\r\n\r\n**Internal QA:**';
                     _.each(internalQAPRMap, (assignees, URL) => {
                         const assigneeMentions = _.reduce(assignees, (memo, assignee) => `${memo} @${assignee}`, '');
-                        issueBody += `\r\n${_.contains(verifiedOrNoQAPRs, URL) ? '- [x]' : '- [ ]'} `;
+                        issueBody += `\r\n${_.contains(resolvedInternalQAPRs, URL) ? '- [x]' : '- [ ]'} `;
```

Comment:
Is it possible that one of these internalQA PRs is verified but not included in `resolvedInternalQAPRs`?

Reply:
Here I guess we could have used also word `verified`,  it is just that for deploy blockers we use resolved and in the end, the meaning is the same.

We will get the `resolvedInternalQAPRs` parameter from here:
https://github.com/Expensify/App/blob/b82ca7cc5d2621654370789c0618eec34d463daf/.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.js#L138

That comes from this call:
https://github.com/Expensify/App/blob/b82ca7cc5d2621654370789c0618eec34d463daf/.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.js#L125-L129

Which comes from here:
https://github.com/Expensify/App/blob/0997c5fc75919e7043c002133c4b0bd9307944f4/.github/libs/GithubUtils.js#L112

And that is ultimately parsed from the issue body in this method:
https://github.com/Expensify/App/blob/0997c5fc75919e7043c002133c4b0bd9307944f4/.github/libs/GithubUtils.js#L176-L200

This is what I meant it is hard to follow 😄 The internalQA PR should not be part of the `verifiedOrNoQAPRs`but if it is resolved/verified (ie the checkbox was ticked), it will be part of the `resolvedInternalQAPRs`.